tests/integration/page-objects/wallet.page.ts

```diff
@@ -85,6 +85,10 @@ export class WalletPage {
     await this.page.waitForSelector(this.homePage, { timeout: 30000 });
   }
 
+  async waitForLoginPage() {
+    await this.page.waitForSelector('text=Sign in with Secret Key', { timeout: 3000 });
```

Comment:
This should probably be a const? or data-test attr?

Reply:
Sure, I will fix it